pkg/remote/remote.go

```diff
@@ -98,7 +98,7 @@ RUN \
   {{range $key, $path := .Caches }}--mount=type=cache,target={{$path}} {{end}}\
   --mount=type=secret,id=known_hosts --mount=id=remote,type=ssh \
   mkdir -p $HOME/.ssh && echo "UserKnownHostsFile=/run/secrets/known_hosts" >> $HOME/.ssh/config && \
-  /okteto/bin/okteto remote-run {{ .Command }} --log-output=json --server-name="${{ .InternalServerName }}" {{ .CommandFlags }}
+  /okteto/bin/okteto remote-run {{ .Command }} --log-output=json --server-name="${{ .InternalServerName }}" {{ .CommandFlags }} || true
```

Comment:
@andreafalzetti with this, we won't return a failure to the user, right? we have to keep that behavior.
Maybe something like `|| echo "something to identify command failed" can be used to populate the error to the user

Reply:
> @pchico83 the change makes the docker run step NOT to fail, but the Okteto CLI will continue to fail because `remote-run` continues to log and the build logger parses the output like is today, so the experience is the following:
> 
> <img alt="Screenshot 2024-06-12 at 15 05 34" width="745" src="https://private-user-images.githubusercontent.com/2318450/338977845-6ed106a9-c901-4d62-badf-a34c6ec3b924.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTgxOTk5NDgsIm5iZiI6MTcxODE5OTY0OCwicGF0aCI6Ii8yMzE4NDUwLzMzODk3Nzg0NS02ZWQxMDZhOS1jOTAxLTRkNjItYmFkZi1hMzRjNmVjM2I5MjQucG5nP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI0MDYxMiUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNDA2MTJUMTM0MDQ4WiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9OTFiMGJmN2Y4ZmU2ZTU1Y2U0MDg1Mzc2MWNjZDUyYTAzZWY0Y2Q3OGRhNjRiZGJjMmJkMWVjNGMxMjYyNjMwNiZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QmYWN0b3JfaWQ9MCZrZXlfaWQ9MCZyZXBvX2lkPTAifQ.6rE27k5GT3v8OnQtGCLsST7rKtp6lhO3UkjXF24b2m8">
> Have a try at the branch if you can. Also the integration test showcase this behaviour

@andreafalzetti oh perfect, thanks for the update!